packages/cozy-clisk/src/launcher/saveFiles.js

```diff
@@ -496,20 +572,10 @@ function getFilePath({ file, entry, options }) {
   if (file) {
     return folderPath + '/' + getAttribute(file, 'name')
   } else if (entry) {
-    return folderPath + '/' + getFileName(entry)
+    return folderPath + '/' + getFileName(entry, options)
   }
 }
 
 function getAttribute(obj, attribute) {
   return get(obj, `attributes.${attribute}`, get(obj, attribute))
 }
-
-async function getOrCreateDestinationPath(client, entry, saveOptions) {
-  const subPath = entry.subPath || saveOptions.subPath
-  let finalPath = saveOptions.folderPath
-  if (subPath) {
-    finalPath += '/' + subPath
-    await client.collection('io.cozy.files').createDirectoryByPath(finalPath)
-  }
-  return finalPath
-}
```

Comment:
Should be part of this commit? It seems that we use `getOrCreateDestinationPath` that is undefined now?